src/pretalx/common/permissions.py

```diff
@@ -0,0 +1,9 @@
+from django.http import Http404
+from rules.contrib.views import PermissionRequiredMixin
+
+
+class PermissionRequired(PermissionRequiredMixin):
```

Comment:
does this  form mixin belong in `permissions.py`? This file seems to be reserved for predicates and rules in all other cases.

Reply:
It's a view mixin; I'll move it to `common.views`.

Reply:
Things in `pretalx.common` usually contain more generalized forms of things – in `common.views`, there are mixins for views (or very generic base classes), but in `common.models`, we have one actual model. I'd be open to a `pretalx.common` refactoring that moves *all* mixins to `pretalx.common.mixins.views`, `pretalx.common.mixins.forms`, etc., but I don't think it's worth changing in just this PR.